compute/address/address_test.go

```diff
@@ -445,3 +550,81 @@ func TestGetRegionalExternal(t *testing.T) {
 	}
 
 }
+
+func TestAssignStaticAddressToExistingVM(t *testing.T) {
+	ctx := context.Background()
+	var seededRand = rand.New(
+		rand.NewSource(time.Now().UnixNano()))
+	tc := testutil.SystemTest(t)
+	instanceName := "test-instance-" + fmt.Sprint(seededRand.Int())
+	addressName := "test-address-" + fmt.Sprint(seededRand.Int())
+	zone := "us-central1-a"
+	region := "us-central1"
+	buf := &bytes.Buffer{}
+
+	// initiate instance
+	err := createTestInstance(tc.ProjectID, zone, instanceName)
+	if err != nil {
+		t.Errorf("createTestInstance got err: %v", err)
```

Comment:
If we fail to create the image, further testing cannot proceed. this should be `t.Fatalf()` instead. same for the IP reservation below.

Reply:
fixed, rebased ✌️